pom.xml

```diff
@@ -728,10 +735,16 @@
                         <artifactId>postgresql</artifactId>
                         <version>${postgresql.driver.version}</version>
                     </dependency>
+                    <dependency>
+                        <groupId>org.apache.commons</groupId>
+                        <artifactId>commons-dbcp2</artifactId>
+                        <version>2.8.0</version>
+                    </dependency>
                 </dependencies>
             </dependencyManagement>
         </profile>
-        <!-- If running Java 7, use JRE7 PostgreSQL driver -->
+
+        <!-- Java 7 -->
```

Comment:
Minor point.  If you are willing, we can *remove* all this logic pertaining to Java 7 (and even remove mention of Java 8).  DSpace 7 only supports Java 11 (or above).  It seems we left around this old Java 7/8 profile stuff by accident.  (This isn't required for this PR though, just noting that this entire section can be removed -- we can always remove it later in a follow-up PR)

Reply:
I believe that's done. I removed the profiles and moved the postgres dependency to 3rd party dependencies.

I was a bit confused about there being a merge-conflict, first I re-pulled, rebased and resolved the conflict. Don't think I've broken anything ha ha